platform-api/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitProjectImporter.java

```diff
@@ -132,11 +136,12 @@ public void importSources(FolderEntry baseFolder,
             final String localPath = localPathResolver.resolve((VirtualFileImpl)baseFolder.getVirtualFile());
             final DtoFactory dtoFactory = DtoFactory.getInstance();
             final String location = storage.getLocation();
+            final String repositoryName = location.substring(location.lastIndexOf("/") + 1, location.length() - 4);
```

Comment:
Please add comment with description and example. Why "/"+1 and location.length() - 4...